bridges/snowbridge/pallets/system-v2/src/lib.rs

```diff
@@ -240,12 +214,9 @@ pub mod pallet {
 	}
 
 	impl<T: Config> Pallet<T> {
-		/// Send `command` to the Gateway on the Channel identified by `channel_id`
-		fn send(origin_location: Location, command: Command, fee: u128) -> DispatchResult {
-			let origin = agent_id_of::<T>(&origin_location)?;
-
+		/// Send `command` to the Gateway from a specific origin/agent
+		fn send(origin: AgentId, command: Command, fee: u128) -> DispatchResult {
```

Comment:
Not all polkadot origins have an agent on Ethereum. I prefer we not use the word `Agent` in this pallet. Within the context of the pallet, the origin location is just being reanchored and hashed to a `H256`.

```suggestion
		/// Send `command` to the Gateway
		fn send(origin: H256, command: Command, fee: u128) -> DispatchResult {
```